index.html

```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="styles.css" />
-    <title>Portfolio Luis David</title>
+    <title>Portfolio-Luis David</title>
+    <link rel="icon" href="images/favicon.ico" type="image/x-icon">
   </head>
 
   <body>
```

Comment:
### Your paragraph
![paragraph](https://user-images.githubusercontent.com/93039359/178596730-abc4388a-78b3-4002-84e5-430a81043efe.png)

- [ ] Please make your paragraph bold for the accessibility purpose :+1:

Reply:
solved